memgpt/cli/cli.py

```diff
@@ -440,6 +441,9 @@ def run(
     else:  # create new agent
         # create new agent config: override defaults with args if provided
         typer.secho("\n🧬 Creating new agent...", fg=typer.colors.WHITE)
+
+        # TODO(swooders) complete hack to get the defaults loaded into AgentState properly
```

Comment:
complete hack to get the defaults loaded into AgentState properly

Reply:
I think we can remove `AgentConfig` here?

Reply:
Punted to later PR:
> Remove all references of AgentConfig and replace them with AgentState, which can be looked up from the metadata store